### PR TITLE
Possible to replace a file in the same location. Issue #61

### PR DESCRIPTION
**Pull request by arturoleon**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Needs a test.

Merge conflict needs to be fixed.

**Comment by XhmikosR** [comment]
@arturoleon: ping to rebase and add a test.

File: tasks/lib/compress.js [hunk 1]
```diff
@@ -47,6 +47,13 @@ module.exports = function(grunt) {
         // Ensure the dest folder exists
         grunt.file.mkdir(path.dirname(filePair.dest));
 
+        // Move the src to a temp file in case that the dest is the same file to avoid a corrupt file
+        if(src === filePair.dest){
```
Comment (XhmikosR) [inline tasks/lib/compress.js]: Add a space after `if`s.

**Comment by philippsimon** [comment]
i just also fixed this issue without a temporary file and added also tests:
https://github.com/gruntjs/grunt-contrib-compress/pull/148

**Comment by XhmikosR** [comment]
Closing in favor of #148.

**Pull request closed without merging**